svm/src/transaction_processor.rs

```diff
@@ -56,6 +65,8 @@ use {
     },
 };
 
+const MAX_NUM_TRANSACTIONS_PER_BATCH: usize = 64;
```

Comment:
`MAX` probably isn't the right word; nothing in protocol to limit this.

Reply:
will change to `DEFAULT_NUM_TRANSACTIONS_PER_BATCH`

Reply:
Addressed all of the above points in commit [ecf0f99dd47c39fb9917a112c447ae0b8e7d8dac], please review